test/api/context.js

```diff
@@ -78,8 +78,8 @@ describe('test MLContext.compute', function() {
   const bufferE = new Float32Array(4);
   const expectedC = [2, 2, 2, 2];
   const expectedE = [3, 3, 3, 3];
-  const descX = {type: 'float32', dimensions: [-1, 2]};
-  const descY = {type: 'float32', dimensions: [2, -1]};
+  const descX = {type: 'float32', dimensions: [3, 2]};
```

Comment:
If descX and descX were just invoked once, please delete these definitions, and add them into builder.input() functions